<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>1299</width>
                <height>841</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>MainWindow</string>
        </property>
        <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
        </property>
        <widget class="QWidget" name="centralwidget">
            <widget class="QLabel" name="label">
                <property name="geometry">
                    <rect>
                        <x>0</x>
                        <y>0</y>
                        <width>141</width>
                        <height>21</height>
                    </rect>
                </property>
                <property name="text">
                    <string>下面显示抓的包</string>
                </property>
            </widget>
            <widget class="QPushButton" name="pushButton_start">
                <property name="geometry">
                    <rect>
                        <x>370</x>
                        <y>500</y>
                        <width>99</width>
                        <height>28</height>
                    </rect>
                </property>
                <property name="text">
                    <string>开始</string>
                </property>
            </widget>
            <widget class="QPushButton" name="pushButton_pause">
                <property name="geometry">
                    <rect>
                        <x>590</x>
                        <y>500</y>
                        <width>99</width>
                        <height>28</height>
                    </rect>
                </property>
                <property name="text">
                    <string>暂停</string>
                </property>
            </widget>
            <widget class="QTableWidget" name="tableWidget">
                <property name="geometry">
                    <rect>
                        <x>0</x>
                        <y>20</y>
                        <width>1301</width>
                        <height>441</height>
                    </rect>
                </property>
                <property name="sizePolicy">
                    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                    </sizepolicy>
                </property>
                <property name="mouseTracking">
                    <bool>false</bool>
                </property>
                <property name="focusPolicy">
                    <enum>Qt::TabFocus</enum>
                </property>
                <property name="contextMenuPolicy">
                    <enum>Qt::ActionsContextMenu</enum>
                </property>
                <property name="layoutDirection">
                    <enum>Qt::LeftToRight</enum>
                </property>
                <property name="autoFillBackground">
                    <bool>false</bool>
                </property>
                <property name="frameShape">
                    <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                    <enum>QFrame::Sunken</enum>
                </property>
                <property name="verticalScrollBarPolicy">
                    <enum>Qt::ScrollBarAlwaysOn</enum>
                </property>
                <property name="sizeAdjustPolicy">
                    <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
                </property>
                <property name="alternatingRowColors">
                    <bool>false</bool>
                </property>
                <property name="selectionBehavior">
                    <enum>QAbstractItemView::SelectRows</enum>
                </property>
                <property name="showGrid">
                    <bool>true</bool>
                </property>
                <property name="gridStyle">
                    <enum>Qt::SolidLine</enum>
                </property>
                <property name="sortingEnabled">
                    <bool>false</bool>
                </property>
                <property name="wordWrap">
                    <bool>true</bool>
                </property>
                <property name="cornerButtonEnabled">
                    <bool>true</bool>
                </property>
                <property name="rowCount">
                    <number>0</number>
                </property>
                <property name="columnCount">
                    <number>7</number>
                </property>
                <attribute name="horizontalHeaderVisible">
                    <bool>true</bool>
                </attribute>
                <attribute name="horizontalHeaderCascadingSectionResizes">
                    <bool>true</bool>
                </attribute>
                <attribute name="horizontalHeaderHighlightSections">
                    <bool>true</bool>
                </attribute>
                <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
                    <bool>false</bool>
                </attribute>
                <attribute name="verticalHeaderVisible">
                    <bool>false</bool>
                </attribute>
                <column/>
                <column/>
                <column/>
                <column/>
                <column/>
                <column/>
                <column/>
            </widget>
        </widget>
        <widget class="QMenuBar" name="menubar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>1299</width>
                    <height>26</height>
                </rect>
            </property>
        </widget>
        <widget class="QStatusBar" name="statusbar"/>
    </widget>
    <resources/>
    <connections/>
</ui>
